frontend/javascripts/oxalis/model/sagas/mesh_saga.ts

```diff
@@ -1259,6 +1260,14 @@ function* handleSegmentColorChange(action: UpdateSegmentAction): Saga<void> {
   }
 }
 
+function* handleMeshOpacityChange(action: UpdateMeshOpacityAction): Saga<void> {
+  const { segmentMeshController } = yield* call(getSceneController);
+  const additionalCoordinates = yield* select((state) => state.flycam.additionalCoordinates);
```

Comment:
aren't `additionalCoordinates` already part of the action? however, I don't think, that's good. the mesh opacity should be changed across all additional coordinates. so, it doesn't make sense to encode some additionalCoord in the action.
my suggestion: remove `additionalCoordinates` from the action and instead access the store for the current additionalCoords where necessary.
ideally, it should not be needed at all (I think, it's currently only needed for `hasMesh` because of the internal dictionary structure), but this is probably out of scope for this PR.

Reply:
I hope that @MichaelBuessemeyer and I found a good solution :blush: